src/Text/Pandoc/Writers/AsciiDoc.hs

```diff
@@ -61,18 +61,28 @@ data WriterState = WriterState { defListMarker    :: String
                                , bulletListLevel  :: Int
                                , intraword        :: Bool
                                , autoIds          :: Set.Set String
+                               , asciidoctorDlct  :: Bool
                                }
 
+defaultWriterState :: WriterState
+defaultWriterState = WriterState{
+                                 defListMarker = "::"
+                               , orderedListLevel = 1
+                               , bulletListLevel = 1
+                               , intraword = False
+                               , autoIds = Set.empty
+                               , asciidoctorDlct = False
+                                 }
+
```

Comment:
Just as a matter of formatting style, I think it looks nicer to line up the bottom `}` with the comma. (And perhaps also the top `{`)

Reply:
done